android/sdl_android/src/main/java/com/smartdevicelink/managers/video/VideoStreamManager.java

```diff
@@ -284,25 +357,57 @@ public void onError(String info) {
             return;
         }
-        // regardless of VideoStreamingParameters are specified or not, we should refer to VideoStreamingCapability.
+        processCapabilitiesWithPendingStart(encrypted, parameters);
+    }
+
+    private void configureGlobalParameters(Context context, Class<? extends SdlRemoteDisplay> remoteDisplayClass, boolean encrypted, List<VideoStreamingRange> listOfStreamingRange) {
+        this.context = new WeakReference<>(context);
+        this.remoteDisplayClass = remoteDisplayClass;
+        this.isEncrypted = encrypted;
+        this.majorProtocolVersion = internalInterface.getProtocolVersion().getMajor();
+        if (listOfStreamingRange != null) {
+            this.listOfStreamingRanges = listOfStreamingRange;
+        }
+    }
+
+
+    private void processCapabilitiesWithPendingStart(boolean encrypted, VideoStreamingParameters parameters) {
+        final VideoStreamingParameters params = (parameters == null) ? new VideoStreamingParameters() : new VideoStreamingParameters(parameters);
         if (majorProtocolVersion >= 5) {
             if (internalInterface.getSystemCapabilityManager() != null) {
-                final VideoStreamingParameters params = ( parameters == null) ? new VideoStreamingParameters() : new VideoStreamingParameters(parameters);
                 internalInterface.getSystemCapabilityManager().getCapability(SystemCapabilityType.VIDEO_STREAMING, new OnSystemCapabilityListener() {
                     @Override
                     public void onCapabilityRetrieved(Object capability) {
-                        params.update((VideoStreamingCapability) capability, vehicleMake);    //Streaming parameters are ready time to stream
-                        startStreaming(params, encrypted);
+                        VideoStreamingCapability castedCapability = ((VideoStreamingCapability) capability);
+                        VideoStreamManager.this.originalCapability = castedCapability;
+
+                        params.update(castedCapability, vehicleMake);    //Streaming parameters are ready time to stream
+                        VideoStreamManager.this.parameters = params;
+
+                        if (listOfStreamingRanges != null) {
+                            // filtering
+                            castedCapability.setAdditionalVideoStreamingCapabilities(
+                                    getSupportedCapabilities(
+                                            listOfStreamingRanges,
+                                            castedCapability.getAdditionalVideoStreamingCapabilities()
+                                    )
+                            );
+                        }
+                        AppCapability appCapability = new AppCapability();
+                        appCapability.setVideoStreamingCapability(castedCapability);
+
+                        OnAppCapabilityUpdated onAppCapabilityUpdated = new OnAppCapabilityUpdated();
```

Comment:
```suggestion
                        OnAppCapabilityUpdated onAppCapabilityUpdated = new OnAppCapabilityUpdated();
                        onAppCapabilityUpdated.setAppCapability(appCapability);
```

@kostyaBoss the negotiate list of VideoStreamingCapabilities are not being sent back to the hmi, currently the OnAppCapabilityUpdated rpc is empty